.github/workflows/release.yml

```diff
@@ -131,6 +131,17 @@ jobs:
             }}/providers/${{ env.GCP_WORKLOAD_IDENTITY_PROVIDER }}
       - name: 'Set up Cloud SDK'
         uses: 'google-github-actions/setup-gcloud@v2'
+      - name: Select Tests for AMD64
+        if: ${{ matrix.arch == 'amd64' }}
+        # Run all tests on AMD64
+        run: echo "CONTAINER_TESTS_TO_RUN="" >> $GITHUB_ENV
+      - name: Select Tests for ARM64
+        if: ${{ matrix.arch == 'arm64' }}
+        # We use QEMU to run ARM64 images on AMD64, but .NET Core isn't supported by QEMU,
+        # for now we only run the basic TestEnvironment tests on ARM64.
+        # https://gitlab.com/qemu-project/qemu/-/issues/249
+        # TODO: enable more tests https://github.com/pulumi/pulumi-docker-containers/issues/289
+        run: echo "CONTAINER_TESTS_TO_RUN="-test.run "TestEnvironment"" >> $GITHUB_ENV
```

Comment:
Does this quoting work OK? I feel like it does because Bash is weird, but just double checking/assume you've already thought of this :sweat_smile:

Reply:
Oops, thanks for catching this. I'm clearly too tired today.